File: businesslib/src/main/java/com/wujia/businesslib/event/EventDoorDevice.java
package com.wujia.businesslib.event;

import org.greenrobot.eventbus.Subscribe;
import org.greenrobot.eventbus.ThreadMode;

/**
 * author ：shenbingkai@163.com
 * date ：2019-03-31
 * description ：
 */
public class EventDoorDevice extends AbsEventEntiry<EventDoorDevice> {

    public EventDoorDevice() {

    }

    public EventDoorDevice(IMiessageInvoke<EventDoorDevice> invoke) {
        super(invoke);
    }

    @Subscribe(threadMode = ThreadMode.MAIN)
    public void onEventBus(EventDoorDevice event) {
        invoke(event);
    }
}

File: businesslib/src/main/java/com/wujia/businesslib/event/EventDoorDevice.kt
package com.wujia.businesslib.event

import org.greenrobot.eventbus.Subscribe
import org.greenrobot.eventbus.ThreadMode

/**
 * author ：shenbingkai@163.com
 * date ：2019-03-31
 * description ：
 */
class EventDoorDevice : AbsEventEntiry<EventDoorDevice> {

    constructor() {

    }

    constructor(invoke: IMiessageInvoke<EventDoorDevice>) : super(invoke) {}

    @Subscribe(threadMode = ThreadMode.MAIN)
    override fun onEventBus(event: EventDoorDevice) {
        invoke(event)
    }
}
